app/team.go

```diff
@@ -1207,7 +1207,7 @@ func (a *App) postRemoveFromTeamMessage(c *request.Context, user *model.User, ch
 	return nil
 }
 
-func (a *App) prepareInviteNewUsersToTeam(teamID, senderId string) (*model.User, *model.Team, *model.AppError) {
+func (a *App) prepareInviteNewUsersToTeam(teamID string, senderId string, memberInvite *model.MemberInvite) (*model.User, *model.Team, []*model.Channel, *model.AppError) {
```

Comment:
`teamID string, senderId string`; I don't know if you added the first string on purpose, but it's not required in go to declare a parameter  type if the following one has the same type.

Reply:
ok

Reply:
thanks for the comment! solved